config/matomoWalletsEvents.ts

```diff
@@ -234,43 +226,38 @@ export const walletsMetrics: WalletsMetrics = {
   events: {
     click: {
       handlers: {
-        onClickAmbire: bindEvent('onClickAmbire'),
-        onClickBlockchaincom: bindEvent('onClickBlockchaincom'),
-        onClickBrave: bindEvent('onClickBrave'),
-        onClickCoin98: bindEvent('onClickCoin98'),
-        onClickCoinbase: bindEvent('onClickCoinbase'),
-        onClickExodus: bindEvent('onClickExodus'),
-        onClickGamestop: bindEvent('onClickGamestop'),
-        onClickLedger: bindEvent('onClickLedger'),
-        onClickMathWallet: bindEvent('onClickMathWallet'),
-        onClickMetamask: bindEvent('onClickMetamask'),
-        onClickOperaWallet: bindEvent('onClickOperaWallet'),
-        onClickTally: bindEvent('onClickTally'),
-        onClickTrust: bindEvent('onClickTrust'),
-        onClickWC: bindEvent('onClickWC'),
-        onClickXdefi: bindEvent('onClickXdefi'),
-        onClickZenGo: bindEvent('onClickZenGo'),
+        // This code returns:
+        // onClickAmbire: bindEvent('onClickAmbire'),
+        // onClickBlockchaincom: bindEvent('onClickBlockchaincom'),
+        // more...
+        ...(
+          Object.keys(MATOMO_WALLETS_EVENTS_TYPES) as Array<
+            keyof typeof MATOMO_WALLETS_EVENTS_TYPES
+          >
+        ).reduce<Record<string, () => void>>((accumulator, eventKey) => {
+          if (eventKey.indexOf('onClick') > -1) {
```

Comment:
Alex fixed this behaviour, there is new event for all clicks, please consult with him